.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
I believe technically it is not needed. I am not too familiar with [browserslist](https://github.com/browserslist/browserslist#browserslistrc) but you are correct in that if this was not present it would fall back on using defaults. It does seem like better practice to have this present rather than fall back to the base case.